lib/python/picongpu/utils/picongpu_memory_calculator.py

```diff
@@ -0,0 +1,363 @@
+#!/usr/bin/env python
+
+"""
+This file is part of PIConGPU.
+
+It is supposed to give an estimate for the memory requirement of a PIConGPU
+simulation per accelerator.
+
+Copyright 2018 PIConGPU contributors
```

Comment:
@ax3l is this short license header fine with you
some other python modules use the same short version

Reply:
that's good and very clear for Python.